src/function.ts

```diff
@@ -1,203 +1,148 @@
+        inputScreen.value = replaceOperatorCheck(inputScreen.value);
+      }
+    });
+  }
+};
 
-          if (operationButton.id !== 'equals') {
-              numEquation = 1;
-              operation = operationButton.id;
-              equationField.value = '';
-              answerField.value = input[0];
-              resetFontSize(equationField, answerField);
-          } else {
-              displayResults(equationField, answerField);
-          }
-      });
+// Output Screen Function
+export const outputEvent = (
+  inputScreen: HTMLInputElement,
+  operationButtons: NodeListOf<Element>,
+  outputScreen: HTMLInputElement
+): void => {
+  for (let operationButton of operationButtons) {
+    operationButton.addEventListener('click', () => {
+      if (operationButton.id === 'equals') {
+        let expression: string = inputScreen.value;
+        // Check if any operator is last Character
+        if (
+          operationSymbols.indexOf(`${expression[expression.length - 1]}`) !==
+          -1
+        ) {
+          return;
+        }
+        // Calculation of the equation
+        let equation: string = inputScreen.value;
+        outputScreen.value = eval(equation);
+        // Font Size Change
+        ansFontSizeChange = checkDecreaseFontSize(
+          outputScreen.value,
+          ansOriginalFontSize
+        );
+        outputScreen.style.fontSize = `${ansFontSizeChange}px`;
+      }
+    });
   }
 };
 
+// Delete a Character
 export const backspaceEvent = (
   backButton: Element,
-  equationField: HTMLInputElement
+  inputScreen: HTMLInputElement
 ): void => {
   backButton.addEventListener('click', () => {
-      //This logic here is to slice the string from starting index
-      //to the length of string - 1
-      //to remove the very last character from the string
-      input[numEquation] = input[numEquation].slice(
-          0,
-          input[numEquation].length - 1
-      );
-
-      //If the string is already empty then change it's value to '0'
-      //instead of slicing it further
-      if (input[numEquation] === '') {
-          input[numEquation] = '0';
-      }
-
-      equationField.value = input[numEquation];
+    inputScreen.value = inputScreen.value.slice(
+      0,
+      inputScreen.value.length - 1
+    );
   });
 };
 
-const resetFontSize = (
-  equationField: HTMLInputElement,
-  answerField: HTMLInputElement
-): void => {
-  equationFontSizeChange = equationOriginalFontSize;
-  equationField.style.fontSize = `${equationFontSizeChange}px`;
-  ansFontSizeChange = ansOriginalFontSize;
-  answerField.style.fontSize = `${ansOriginalFontSize}px`;
+// Replace the old Operator with new Operator
+const replaceOperatorCheck = (temp: string): string => {
+  let len: number = temp.length;
+  if (len > 1) {
+    if (
+      operationSymbols.indexOf(`${temp[len - 2]}`) !== -1 &&
+      operationSymbols.indexOf(`${temp[len - 1]}`) !== -1
+    ) {
+      let replacedOperator: string;
+      if (len == 2) {
```

Comment:
why len?

Reply:
variable name len is refactored as lengthOf Temporary